src/components/Card.js

```diff
@@ -20,21 +38,30 @@ export default class Card {
     this._deleteButton = this._element.querySelector(".card__delete-button");
     this._likeButton = this._element.querySelector(".card__like-button");
     this._image = this._element.querySelector(".card__image");
+    this._likeNumber = this._element.querySelector(".card__like-number");
     this._setEventListeners();
     this._image.src = this._link;
     this._image.alt = this._name;
+    this._likeNumber.textContent = this._like;
```

Comment:
The code for rendering likes gets duplicated. Card should have only one public method for updating likes, it should process other logic by itself. Also, it's needed to split the logic for updating likes array and rendering likes.

The code for rendering likes should look like this:

```js
// in Card class 

// call this method from index.js after successful api response in `then` block
setLikes(likes) { 
   // set instance variable
    this._likes = likes; 
    this._renderLikes(); 
} 

isLiked() {
 // return true if user liked the card, otherwise false
}

_renderLikes() {
  //set  likes counter content using this._likes.length

  if (this.isLiked()) {
    // add active class to the like button
  } else {
    // remove active class from the like button
  }
}

createCard() { 
  ...
  // render initial likes, you saved them in the constructor
  this._renderLikes()
}

// index.js
if (card.isLiked()) {
  api.dislikeCard(...).then(response => card.setLikes(response.likes)).catch(() => /*handle error*/)
} else {
 // the same for adding like
}
```